packages/clay-drop-down/src/DropDown.tsx

```diff
@@ -231,6 +238,14 @@ function ClayDropDown<T>({
 				'aria-controls': ariaControls,
 				'aria-expanded': internalActive,
 				'aria-haspopup': 'true',
+				children: (
+					<>
+						{trigger.props.children}{' '}
+						{triggerIcon !== 'undefined' && (
+							<ClayIcon className="ml-1" symbol={triggerIcon} />
+						)}
+					</>
+				),
```

Comment:
I'm still not sure about this yet, we don't have any certainty that the trigger can be a button, so by doing this we are inferring that it will always be a button, which may not be true because the DropDown can be used for different purposes, such as passing an input or other custom element that could break any application.

I think that to avoid problems like this or reduce them, we can create a conditional before passing our `children` with the icon and check if the `trigger` is a valid component and if it is a ClayButton component, so we can be more certain and reduce possible failures in the DXP.

```js
children: React.isValidElement(trigger)
    // @ts-ignore
    && child?.type.displayName === 'ClayButton' ?
        (
       	    <>
          		{trigger.props.children}{' '}
          		{triggerIcon !== 'undefined' && (
         			<ClayIcon className="ml-1" symbol={triggerIcon} />
          		)}
           	</>
    	),
	: undefined,
```